src/service/world-api-adapter.js

```diff
@@ -223,6 +219,24 @@ module.exports = function (config) {
         },
 
         /**
+        * Set the filter for the current instance of the world adapter
+        *
+        * **Example**
+        * var ws = new F.service.World({...}).load('123').addUser({ userId: '123' });
+        *
+        *
+        */
+        load: function (worldId) {
+            if (!worldId || typeof worldId !== 'string') {
+                throw new Error('load needs a worldId string to load (' + worldId + ')');
+            }
+
+            serviceOptions.filter = worldId;
+
+            return this;
```

Comment:
We should do a `get` and return a promise, or call this something else - the `runservice` and `dataservice` have `.load` which behaves differently